docs_source_files/content/webdriver/http_proxies.en.md

```diff
@@ -3,3 +3,51 @@ title: "Http proxies"
 weight: 7
 ---
 
+A proxy server acts as an intermediary for requests between a client and a server. In simple, the traffic flows through the proxy server on its way to the address you requested and back.
```

Comment:
Could you please leave the column width to 70-80 characters?

Reply:
Updated!